_includes/manuals/nightly/3.7_migration.md

```diff
@@ -0,0 +1,185 @@
+
+### Migrating to PostgreSQL
+
+Starting from Foreman 1.25/2.0, MySQL is no longer a supported database
+management system. This chapter describes the process of migrating from MySQL
+to PostgreSQL.
+
+Full backup is absolute necessary before starting the process. We recommend
+both database dump and snapshot of whole operating system and data if possible
+when running in a VM or on a LVM.
+
+#### Prerequisites
+
+It's a good idea to delete unwanted records from the database. The smaller the
+database is, the faster will be the migration process. Typical offenders are
+audit records, facts and reports. Foreman ships with rake tasks and cron job to
+execute these cleanup tasks, but they can be executed manually. Steps are shown
+down below after all instances are stopped.
+
+Foreman database must be cleaned up from any leftovers from uninstalled
+plugins. When a Foreman plugin is activated, database migration is perfored and
+new tables and indices are created. However when a plugin is uninstalled and
+deactivated, no tables are ever dropped and the data is left in the database
+forever. It's good idea to review at least OS installed package log to find out
+if any foreman package was ever uninstalled. Unfortunately, there is no list of
+tables for each plugin.
+
+#### Preparing the environment
+
+This blog post assumes that you already have the source and target databases
+set up. Make sure to setup the Postgresql database with UTF-8 enabled.
```

Comment:
how does one do that? using the installer? imho you need to allow the user to copy-paste, or even write a small script to automate the migration.

Reply:
Yeah I also noticed that later on, I have removed this completely, everything is UTF-8 by default these days. Maybe @timogoebel remember why he added that?